tools/python_api/src_py/connection.py

```diff
@@ -128,21 +128,21 @@ def execute(
             raise RuntimeError(msg)  # noqa: TRY004
 
         if len(parameters) == 0 and isinstance(query, str):
-            _query_result = self._connection.query(query)
+            query_result = self._connection.query(query)
         else:
             prepared_statement = self._prepare(query, parameters) if isinstance(query, str) else query
-            _query_result = self._connection.execute(prepared_statement._prepared_statement, parameters)
-        if not _query_result.isSuccess():
-            raise RuntimeError(_query_result.getErrorMessage())
-        current_query_result = QueryResult(self, _query_result)
-        if not _query_result.hasNextQueryResult():
+            query_result = self._connection.execute(prepared_statement._prepared_statement, parameters)
+        if not query_result.isSuccess():
+            raise RuntimeError(query_result.getErrorMessage())
+        current_query_result = QueryResult(self, query_result)
+        if not query_result.hasNextQueryResult():
             return current_query_result
         all_query_results = [current_query_result]
-        while _query_result.hasNextQueryResult():
-            _query_result = _query_result.getNextQueryResult()
-            if not _query_result.isSuccess():
-                raise RuntimeError(_query_result.getErrorMessage())
-            all_query_results.append(QueryResult(self, _query_result))
+        while query_result.hasNextQueryResult():
+            query_result = query_result.getNextQueryResult()
+            if not query_result.isSuccess():
+                raise RuntimeError(query_result.getErrorMessage())
+            all_query_results.append(QueryResult(self, query_result))
```

Comment:
I would not change this part. I feef like it is better to keep it `_query_result` to distinguish the native `query_result` exposed from C++ side from the Python query result wrapper.